src/charts/tooltip.js

```diff
@@ -405,8 +406,12 @@ define(function(require){
         function updateTitle(dataPoint) {
             let tTitle = title;
 
```

Comment:
Let's compute the date only one time:
  let formattedDate = formatDate(new Date(dataPoint[dateLabel]));
  ...

Reply:
Duh... whoops! Updated in 24f951e4c0fe68860ff301452ffdd01c6960b41d.